.github/workflows/lint.yml

```diff
@@ -145,6 +160,8 @@ jobs:
           architecture: x64
       - name: Checkout PyTorch
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # to allow tools/clang_tidy.py to do its thing
```

Comment:
is "do its thing" accessing git history?

Reply:
yep: https://github.com/pytorch/pytorch/blob/cc11aaaa60aadf28e3ec278bce26a42c1cd68a4f/tools/clang_tidy.py#L109-L137